website/docs/troubleshooting/troubleshooting-podman.md

```diff
@@ -233,3 +233,24 @@ This might appear when either:
 3. Restart the Podman machine to recreate and activate the default Docker socket path.
 
 _Note:_ If Docker Desktop is started again, it will automatically re-alias the default Docker socket location and the Podman compatibility warning will re-appear.
+
+## Uninstalling Podman Desktop preserves the old configuration data  
+
+#### Issue
+
+After uninstalling Podman Desktop, the configuration data persists even though it is not needed for a fresh installation.
+
+
+#### Solution
+
+**_Deleting Podman Desktop configuration_**
+
+- Go to the `~/.local/share/containers/podman-desktop/` directory and delete the `configuration` folder.
+
+**_Deleting Podman configuration_**  
```

Comment:
I think it would also be nice to indicate where Podman is storing its files.
`$HOME/.config/containers`